common/executor.js

```diff
@@ -30,9 +31,6 @@ module.exports = function (callback) {
       _setExecutorAsReqProc.bind(null, bag)
     ],
     function () {
-      if (bag.statusPoll)
```

Comment:
This is moved to immediately after the task execution so that it's as close to end of task execution as possible.